workers/unity/Packages/io.improbable.gdk.debug/.codegen/Source/Generators/DebugExtensions/ComponentVisualElementGenerator.cs

```diff
@@ -22,28 +29,184 @@ public static CodeWriter Generate(UnityComponentDetails details)
                     {
                         type.Line($"public override ComponentType ComponentType {{ get; }} = ComponentType.ReadOnly<{details.Name}.Component>();");
 
+                        type.TextList(TextList.New(details.FieldDetails.Select(ToFieldDeclaration)));
```

Comment:
Might be worth adding a variant of TypeBlock.TextList that takes an IEnumerable and does TextList.New for you under the hood?

Reply:
Yeah that would be a bit more ergonomic. There's an `ICollection` version I think, but not `IEnumerable`